collections/landsat-8-l2.yaml

```diff
@@ -147,5 +147,227 @@ CubeDimensions:
+    - 'http://www.opengis.net/def/crs/EPSG/0/32706'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32707'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32708'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32709'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32710'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32711'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32712'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32713'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32714'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32715'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32716'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32717'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32718'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32719'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32720'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32721'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32722'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32723'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32724'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32725'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32726'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32727'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32728'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32729'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32730'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32731'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32732'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32733'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32734'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32735'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32736'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32737'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32738'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32739'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32740'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32741'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32742'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32743'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32744'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32745'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32746'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32746'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32748'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32749'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32750'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32751'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32752'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32753'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32754'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32755'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32756'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32757'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32758'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32759'
+    - 'http://www.opengis.net/def/crs/EPSG/0/32760'
+    - 'http://www.opengis.net/def/crs/SR-ORG/0/98739'
+  eo:gsd:
+    - 15
```

Comment:
[Landsat 8 L2](https://docs.sentinel-hub.com/api/latest/data/landsat-8-l2/#available-bands-and-data) doesn't have the band B08 which is the only one in [L1](https://docs.sentinel-hub.com/api/latest/data/landsat-8/#available-bands-and-data) that has 15m spatial resolution.